dashboard/src/app/workspaces/workspace-details/environments/environments.controller.ts

```diff
@@ -83,13 +84,14 @@ export class WorkspaceEnvironmentsController {
     this.cheRecipeService = cheRecipeService;
 
     this.editorOptions = {
-      lineWrapping: true,
-      lineNumbers: false,
+      wordWrap: 'on',
+      lineNumbers: 'on',
       readOnly: true,
-      gutters: [],
+      lineDecorationsWidth: 0,
+      lineNumbersMinChars: 0,
       onLoad: (editor: any) => {
         $timeout(() => {
-          editor.refresh();
+          editor.render(true);
```

Comment:
We added this timeout depends on the CodeMirror bug. I think we can remove it at all (in all parts of the code).
WDYT?

Reply:
Let it be a separate issue.